### PR TITLE
CIV-13069 optional parts on credit hire section

### PR DESCRIPTION
**Pull request by Omaira-Melo-Hmcts**
### JIRA link (if applicable) ###

[Making parts of the credit hire order an optional choice ](https://tools.hmcts.net/jira/browse/CIV-13069)

### Change description ###

Making parts of the credit hire order an optional choice

From slide 6 of List of amendments for SDO.

The following parts of the credit hire order need to be optional:
CreditHire_WithDetails

The changes are to be made for both (Standard Direction Order and Direction Order (Judgment):

Fast track order
A trial to decide the amount of damages order

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by sjamihmcts — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json:1]: Shouldn't this be in a SDO-R2 file?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-R2-CUI-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-R2-CUI-nonprod.json:1]: Shouldn't this be in a SDO-R2 file?

File: ccd-definition/CaseEventToFields/DefaultJudgmentSDO-DJ-prod.json [hunk 1]
```diff
@@ -174,5 +162,17 @@
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "N",
     "FieldShowCondition": "hearingMethodValuesTrialHearingDJ = \"DONOTSHOW\""
+  },
```
Comment (hmcts-version1-ignacio) [inline ccd-definition/CaseEventToFields/DefaultJudgmentSDO-DJ-prod.json:165]: Is this a change?

File: ccd-definition/CaseField/CaseFieldSDO-DJ-prod.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "CaseTypeID": "CIVIL",
+    "ID": "trialCreditHire",
+    "Label": "Credit hire",
+    "FieldType": "TrialCreditHire",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  }
+]
+
+
```
Comment (hmcts-version1-ignacio) [inline ccd-definition/CaseField/CaseFieldSDO-DJ-prod.json:12]: Extra lines?

File: ccd-definition/ComplexTypes/DefaultJudgmentSDO/TrialCreditHireDetails-SDO-DJ-R2-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/ComplexTypes/DefaultJudgmentSDO/TrialCreditHireDetails-SDO-DJ-R2-nonprod.json:1]: Shouldn't it be "...-DJ-SDO-R2-prod.json"?

File: ccd-definition/ComplexTypes/DefaultJudgmentSDO/TrialCreditHireMain-SDO-DJ-R2-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/ComplexTypes/DefaultJudgmentSDO/TrialCreditHireMain-SDO-DJ-R2-nonprod.json:1]: Shouldn't it be "...-DJ-SDO-R2-prod.json"?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-R2-CUI-nonprod.json
Comment (Omaira-Melo-Hmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-R2-CUI-nonprod.json:1]: Like this ? AuthorisationCaseFieldSDO-R2-DJ-CUI-nonprod

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json
Comment (Omaira-Melo-Hmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json:1]: I don't think so, cause @bhagyashreesharma90 advice me only add the nonprod to the existing name. What do you think @bhagyashreesharma90 ?

**Review by ShwetaTandel-hmcts — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json
Comment (bhagyashreesharma90) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json:1]: @Omaira-Melo-Hmcts , Sorry for confusion but I think adding SDOR2 is better option so when we go live, we know which ccd file we need to change from nonprod to prod.

Comment (Omaira-Melo-Hmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldSDO-DJ-nonprod.json:1]: Great ! I'll rename it ! Thank you @bhagyashreesharma90 and @hmcts-version1-ignacio  for you comments.

**Review by sjamihmcts — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by bhagyashreesharma90 — APPROVED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by sjamihmcts — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Pull request merged**